src/index.ts

```diff
@@ -55,41 +50,36 @@ function parseArguments() {
     console.log('Launching BiDi server.');
 
     const args = parseArguments();
-
     const bidiPort = args.port;
-    const browserExecutablePath = args.browser || process.env.BROWSER_PATH;
     const headless = args.headless !== 'false';
 
-    BidiServerRunner.run(
-      bidiPort,
-      (bidiServer: IServer) => {
-        return _onNewBidiConnectionOpen(
-          browserExecutablePath,
-          headless,
-          bidiServer
-        );
-      },
-      _onBidiConnectionClosed
-    );
+    BidiServerRunner.run(bidiPort, (bidiServer: IServer) => {
+      return _onNewBidiConnectionOpen(headless, bidiServer);
+    });
     console.log('BiDi server launched.');
   } catch (e) {
     console.log('Error', e);
   }
 })();
 
+/**
+ * @returns delegate to be called when the connection is closed
+ */
 async function _onNewBidiConnectionOpen(
-  browserExecutablePath: string,
   headless: boolean,
   bidiServer: IServer
-): Promise<BrowserProcess> {
-  // Launch browser.
-  const browserProcess = await launchBrowser(browserExecutablePath, headless);
+): Promise<() => void> {
+  // Hijack Puppeteer's implementation of fetching and launching browser.
```

Comment:
What do you mean by “hijack”? What does “fetching” refer to here? (`launch()` doesn’t do fetching.)

If we need more control about which Chromium revision to download we could use `puppeteer.createBrowserFetcher()` in the future, but for now using the bundled-with-Puppeteer Chromium seems fine 👍 .